lib/Conversion/TorchToTcp/Elementwise.cpp

```diff
@@ -380,6 +388,66 @@ class ConvertAtenUnaryOp : public OpConversionPattern<AtenOpT> {
   }
 };
 
+class ConvertOperatorAtanOp : public OpConversionPattern<OperatorOp> {
```

Comment:
As discussed offline, we should add the `torch.atan` op first in TorchMLIR, then follow it up with it in TCP, like any other unary op. These shouldn't be handled as opaque operators.